docs/modules/dqn.rst

```diff
@@ -27,7 +27,16 @@ and its extensions (Double-DQN, Dueling-DQN, Prioritized Experience Replay).
 Notes
 -----
 
-- Original paper: https://arxiv.org/abs/1312.5602
+- DQN paper: https://arxiv.org/pdf/1312.5602.pdf
```

Comment:
This could link to landing page rather than directly to PDF, i.e. https://arxiv.org/abs/1312.5602